src/installation/live-images/prep.md

```diff
@@ -30,16 +46,25 @@ Write the live image to the device:
 # dd bs=4M if=/path/to/void-live-ARCH-DATE-VARIANT.iso of=/dev/sdX
 ```
 
-Ensure all data is flushed before disconnecting the device:
+dd won't print anything until it's completed (or if it failed), so depending on
+the device, this can take a few minutes or longer.
+
+Finally, ensure all data is flushed before disconnecting the device:
 
 ```
 $ sync
+90+0 records in
```

Comment:
This isn't the output from sync, please don't imply it is here.

Reply:
That wasn't supposed to be in that spot. My bad.